src/main/java/com/puppycrawl/tools/checkstyle/checks/TranslationCheck.java

```diff
@@ -537,7 +537,7 @@ private void logIoException(IOException exception, File file) {
          * @param path common path of files which are included in the resource bundle.
          * @param extension common extension of files which are included in the resource bundle.
          */
-        ResourceBundle(String baseName, String path, String extension) {
+        /* default */ ResourceBundle(String baseName, String path, String extension) {
```

Comment:
public,

in private inner classes, it is ok to make public ctors.
Please apply such rule to all others.

Reply:
if class is private, inner ctor needs to also be private.
class can't leave itself since it is private. only reason a ctor could be public is if reflection is used somehow.